src/main/java/io/appium/java_client/ios/touch/DoubleTapOptions.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.ios.touch;
+
+import io.appium.java_client.touch.OptionsWithAbsolutePositioning;
```

Comment:
If this class was added only for bacward compatibility so I think it might be better to make it nested in `IOSTouchAction`

Reply:
but this should be visible for TouchAction options as well.